mysql/tests/test_mysql.py

```diff
@@ -42,14 +42,20 @@ def test_complex_config(aggregator, instance_complex):
     mysql_check = MySql(common.CHECK_NAME, {}, {}, instances=[instance_complex])
     mysql_check.check(instance_complex)
 
-    # Test service check
-    aggregator.assert_service_check('mysql.can_connect', status=MySql.OK, tags=tags.SC_TAGS, count=1)
+    _assert_complex_config(aggregator, count=1)
 
-    aggregator.assert_service_check('mysql.replication.slave_running', status=MySql.OK, tags=tags.SC_TAGS, at_least=1)
 
-    ver = map(lambda x: int(x), mysql_check.mysql_version[mysql_check._get_host_key()])
-    ver = tuple(ver)
+@pytest.mark.e2e
+def test_e2e(dd_agent_check, instance_complex):
+    aggregator = dd_agent_check(instance_complex, rate=True)
```

Comment:
if you dont set rate=True, you dont need to make count a parameter, right?

Reply:
Right, not needed in this case since the rate metrics are optionally asserted. So leaving to count=1 should not have impact.

Fixed.